src/SimpleThings/EntityAudit/EventListener/CreateSchemaListener.php

```diff
@@ -109,7 +109,7 @@ public function postGenerateSchema(GenerateSchemaEventArgs $eventArgs)
             'autoincrement' => true,
         ));
         $revisionsTable->addColumn('timestamp', 'datetime');
-        $revisionsTable->addColumn('username', 'string');
+        $revisionsTable->addColumn('username', 'string')->setNotnull(false);
```

Comment:
why it is a BC break? Allow NULL isn't a BC break and your application will not break down.